migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc

```diff
@@ -1,11 +1,11 @@
 [id="planning-considerations-3-4"]
-= Planning considerations
+= Network considerations
 include::modules/common-attributes.adoc[]
 :context: planning-considerations-3-4
 
 toc::[]
 
-This section describes considerations and strategies for planning your migration from {product-title} 3 to 4.
+This section explains how to preserve your applications' network connections after the migration from {product-title} 3 to 4.
```

Comment:
We try to avoid constructions like "This section explains" because it is anthropomorphic and not user-centric.

Suggestion: Review the strategies for updating redirecting your application network traffic after migration.